src/components/Table.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState, useEffect } from 'react';
+import './Table.css';
+
+export default function Table() {
+  const [items, setItems] = useState([]);
+
+  useEffect(() => {
+    getData();
+  });
+
+  async function getData() {
+    const curated = await fetch(
+      `https://api.pexels.com/v1/curated?page=1&per_page=${process.env.REACT_APP_PER_PAGE}`,
```

Comment:
We should keep base URL as environment variable as well. And make a new file (e.g. `config.js`) that will contain all such variables initialization